pkgs/development/python-modules/gremlinpython/default.nix

```diff
@@ -66,13 +68,26 @@ buildPythonPackage rec {
   disabledTestPaths = [
```

Comment:
We should switch that over to pytestCheckHook

Reply:
?